src/confluent_kafka/src/Consumer.c

```diff
@@ -206,6 +227,35 @@ static PyObject *Consumer_unsubscribe (Handle *self,
 }
 
 
+static PyObject *Consumer_incremental_assign (Handle *self, PyObject *tlist) {
+
+	rd_kafka_topic_partition_list_t *c_parts;
+	rd_kafka_error_t *error;
+
+        if (!self->rk) {
```

Comment:
mix of whitespace and tabs, stick to ws for new code.
Sorry!

Reply:
it goes further, please also be sorry about 8 space tabs